tools/d2g/d2g.go

```diff
@@ -568,10 +568,6 @@ func envMappings(dwPayload *dockerworker.DockerWorkerPayload) string {
 		"TASK_ID",
 	}
 
-	if dwPayload.Features.TaskclusterProxy {
-		additionalEnvVars = append(additionalEnvVars, "TASKCLUSTER_PROXY_URL")
-	}
-
```

Comment:
I think I would prefer having any and all env vars be added directly in this helper `envMappings()` function. So this could change to something like:
```go
if dwPayload.Features.TaskclusterProxy {
    additionalEnvVars = append(additionalEnvVars, "TASKCLUSTER_PROXY_URL=${TASKCLUSTER_PROXY_URL}/localhost/taskcluster")
}
```

Reply:
That doesn't work, because additionalEnvVars is then passed through `shell.Escape` (via `envSetting`).

Reply:
Also, the `/localhost/taskcluster` does need to be within the braces.

Reply:
Ah right, thanks, I had forgotten about `shell.Escape`.